crates/codegen/runner/src/main.rs

```diff
@@ -0,0 +1,151 @@
+mod passes;
+
+use anyhow::Result;
+use codegen_language_definition::model::Language;
+use codegen_runtime_generator::RuntimeGenerator;
+use codegen_spec::Spec;
+use codegen_testing::TestingGeneratorExtensions;
+use infra_utils::cargo::CargoWorkspace;
+use infra_utils::codegen::CodegenFileSystem;
+use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
+use solidity_language::SolidityDefinition;
+use testlang_language::TestlangDefinition;
+
+use crate::passes::generate_passes;
+
+fn main() {
+    // TODO(#1286): this is a poor man's version of a "build script" that runs all codegen steps in parallel.
+    // It is temporary step, and should be simplified/removed in subsequent PRs.
```

Comment:
I'm curious, how do you imagine this in the future?

Reply:
At least for the next step, we discussed removing `testlang`, which will also remove the need to have a separate runtime or even stubs/product separation.